client/src/components/NewsArticle/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import './style.scss';
+import { Layout } from '../Layout';
+import { fetchNewsData } from '../../containers/NewsLogic/action';
+import { connect } from 'react-redux';
+import Loader from 'react-loader-spinner';
+
+
+class NewsArticle extends Component {
+  componentDidMount() {
+    const { fetchNewsData } = this.props;
+
+    fetchNewsData();
+  }
+  
+  shortText(text) {
+    return text.length < 100 ? text
+      : <h2>{text.slice(0, 80)}...</h2>;
+  };
+
+  render() {
+    const { newsData, match } = this.props;
+
+    const { id } = match.params;
+    const art = newsData.find(art => id === art._id);
+
+    return (
+      !newsData.length ? (
+        <div className='data-loading'>
+          <Loader
+            type='TailSpin' color='#C63638'
+            height='200' width='200'
+          />
+        </div>
+      ) : (
+        <Layout>
+          <section className='single-news-container'>
+            <div className='news-wrapper'>
+              <div className='top-info'>
+                <h4>
```

Comment:
Why do you hard code all this text? It should come from back, shouldn't it?

Reply:
this is my mistake)